packages/@okta/vuepress-site/docs/reference/api/authenticators-admin/index.md

```diff
@@ -1073,4 +1145,145 @@ The Authenticator object defines the following properties:
         }
     }
 }
-```
\ No newline at end of file
+```
+
+### Authenticator Method object
+
+#### Authenticator Method properties
+
+| Property | Type | Description  | Applies to Authenticator Method type |
+| -------- | ---- | ------------ | ------------------------------------ |
+| `status` | String (Enum) | The status of the authenticator method. Possible values: `ACTIVE` or `INACTIVE` | All Authenticator Methods |
+| `type` | String (Enum) | The type of authenticator method. Possible values: `cert`, `duo`, `email`, `idp`, `otp`, `password`, `push`, `security_question`, `signed_nonce`, `sms`, `totp`, `voice`, or `webauthn`  |  All Authenticator Methods |
+| `_links` | [JSON HAL](https://tools.ietf.org/html/draft-kelly-json-hal-06) | Link relations for this Authenticator Method object  |  All Authenticator Methods |
+| `settings` | [Authenticator Method Settings object](#authenticator-method-settings-object) | Specific settings for the authenticator method  | `otp`, `push`, `signed_nonce`, `totp`, `webauthn` |
+
+### Authenticator Method Settings object
+
+#### Authenticator Method Settings properties
+
+| Property | Type | Description  | Applies to Authenticator Method type |
+| -------- | ---- | ------------ | ------------------------------------ |
+| `algorithms` | String (Enum) | Algorithms supported. Possible values: `RS256`, `ES256`, `HMACSHA1`, `HMACSHA256`, or `HMACSHA512` | `otp`, `totp`, `push`, `signed_nonce` |
+| `attachment` | String (Enum) | Method attachment. Possible values: `ANY`, `BUILT_IN`, or `ROAMING`  | `webauthn` |
+| `keyProtection` | String (Enum) | Indicates whether you must use a hardware key store for `push` and `signed_nonce` methods or if any type of key store is allowed. Supported values: `ANY` or `HARDWARE`  | `push`, `signed_nonce`|
+| `protocol` | String (Enum) | The protocol used. Possible values: `SYMANTEC`,`TOTP`, or `YUBICO` | `otp` |
+| `timeIntervalInSeconds` | Integer | Time interval for TOTP in seconds | `totp` |
+| `encoding` | String (Enum) | The shared secret encoding. Possible values: `Base32`, `Base64`, or `Hexadecimal`  | `totp` |
+| `passCodeLength` | Integer | Number of digits in an OTP value  | `otp`, `totp` |
+| `acceptableAdjacentIntervals` | Integer | Also known as the clock drift interval, this setting allows you to build in tolerance for any time difference between the token and the server. Number of acceptable adjacent intervals. For example, with a `timeIntervalInSeconds` of 60 seconds and an `acceptableAdjacentIntervals` value of 5, the result of 60 X 5 is 300, which means that Okta accepts passcodes within 300 seconds before or after the end user enters their code. Possible values: 0&ndash;10 | `otp` |
```

Comment:
nit: Could maybe move the "Number of acceptable adjacent intervals." bit to be first?

i.e. Number of acceptable adjacent intervals, also known as the clock drift interval.  This setting allows you to build in tolerance for any time difference between the token and the server.  For example, with a `timeIntervalInSeconds` of 60 seconds and an `acceptableAdjacentIntervals` value of 5, the result of 60 X 5 is 300, which means that Okta accepts passcodes within 300 seconds before or after the end user enters their code. 

Might make more sense to keep it as-is if we're trying to be consistent with else where though.

Reply:
Opps, I forgot to remove the "Number of acceptable adjacent intervals. " sentence in the rewrite (which was just from the property name). Your suggestion sounds good. I'll use it 👍